atom/browser/api/atom_api_web_contents.cc

```diff
@@ -421,6 +422,29 @@ bool WebContents::HandleContextMenu(const content::ContextMenuParams& params) {
   return true;
 }
 
+void WebContents::FindReply(content::WebContents* web_contents,
+                            int request_id,
+                            int number_of_matches,
+                            const gfx::Rect& selection_rect,
+                            int active_match_ordinal,
+                            bool final_update) {
+  v8::Locker locker(isolate());
+  v8::HandleScope handle_scope(isolate());
+
+  mate::Dictionary result = mate::Dictionary::CreateEmpty(isolate());
+  if (number_of_matches == -1) {
+    result.Set("requestId", request_id);
+    result.Set("selectionArea", selection_rect);
+    result.Set("finalUpdate", final_update);
+    Emit("find-in-page-response", result);
```

Comment:
`FindReply` is notified several times during a request and of these only two situations matter for us, one when `number_of_matches` is -1 (when found atleast one) and other when `finalUpdate` is true, indicates final response for the request with total number of matches. So wanted to emit the event only under these two scenarios and ignore others.